data/json/items/containers.json

```diff
@@ -112,7 +112,10 @@
         "to_hit":-1,
         "material":"steel",
         "contains":1,
-        "rigid": true
+        "rigid": true,
+        "watertight": true,
+        "seals": true,
+        "qualities": [["CONTAIN", 1]]
```

Comment:
Adding this quality is good. All the rest needs to go; it's a bowl, not a bottle.